### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by akanksha2300**
Please merge. Added test comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
 # Mindshift
 
+# test change AA
+
 ```
 pip install -r requirements.txt
 ```
 
 For Idling Detection
+
```
Comment (Manjimadutta) [inline README.md:10]: Good till now.

**Review by Manjimadutta — APPROVED** [review]
The changes she made is looking good so far.

File: README.md (hunk 1 above)
Comment (sanjmeh) [reply README.md:10]: FIne. Go ahead.

**Review by Manjimadutta — APPROVED** [review]
All well.

**Pull request merged**